agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptions.java

```diff
@@ -192,6 +194,12 @@
 	 */
 	/* package */ SapNwdiApplications sapNetWeaverJavaApplications = null;
 
+	/**
+	 * Whether to obfuscate security related configuration options when dumping them
+	 * into the log or onto the console or not.
+	 */
+	/* package */ boolean obfuscateSecurityRelatedOutputs = true;
```

Comment:
Any reason this has package visibility? Given that you define a getter below, I'd assume this can be `private`.

Reply:
(1.) For consistency reasons: All others have this visibility, tool. And (2.), because `AgentOptionsParser` is accessing it (as all the other options).